src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/PaidInFullStaffNotificationService.java

```diff
@@ -15,6 +16,7 @@
 import java.util.Map;
 
 @Service
+@ConditionalOnProperty("feature_toggles.staff_emails_enabled")
```

Comment:
could have done on handler level as the service is used only on the handler. The new instance of handler won't be created then in first place.